pkg/service/restore/service_restore_integration_test.go

```diff
@@ -1071,6 +1100,9 @@ func restoreWithVersions(t *testing.T, target Target, keyspace string, loadCnt,
 	for _, tc := range secondCorrupt {
 		swapWithNewest(tc, tag5)
 	}
+	for _, tc := range keys(secondCorruptCrc32) {
+		swapWithNewest(tc, tag5)
+	}
```

Comment:
This also wouldn't be needed, if .crc32 files were a part of first/second/bothCorrupt.

Reply:
let's leave it